canonical-web-and-design/maas-ui
# Update vanilla-framework to v2.33.0 and react-components to v0.20.0

## Done

- Update vanilla-framework to v2.33.0 and react-components to v0.20.0
- Updated custom notification group styling in react and legacy apps

## QA

- Run `yarn start` to get the full UI running
- Create some fake notifications through the api or alternatively create a new user in `bolla` and log in as them (for some reason new users have a bunch of random notifications)
- Check that the notifications look ok and they work as before
- Click around the UI in both the legacy and react apps and check that nothing is broken

## Fixes

Fixes #2910 
Fixes: #2888.

## Screenshot
### Server notifications w/ grouping
![Screenshot 2021-07-30 at 13-46-33 Machines bolla MAAS](https://user-images.githubusercontent.com/25733845/127597309-6c0b0210-6d08-403f-b8e3-297a2b29be1f.png)


### In-page notifications
![Screenshot 2021-07-30 at 13-47-38 test maas network bolla MAAS](https://user-images.githubusercontent.com/25733845/127597330-4906dc07-1c62-43e3-a2f7-73c6f98eb9b5.png)


### Legacy notifications
![Screenshot 2021-07-30 at 13-47-00 Controllers bolla MAAS](https://user-images.githubusercontent.com/25733845/127597320-56fc8687-8844-4e1a-9abc-d948f5e68ee9.png)
